wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ConfigurationMessageCell/ConversationMessageActionController.swift

```diff
@@ -177,12 +179,12 @@ final class ConversationMessageActionController {
     }
 
     var doubleTapAction: MessageAction? {
-        return message.canBeLiked ? .like : nil
+        return message.canAddReaction ? .like : nil
     }
 
     // MARK: - Handler
 
-    private func perform(action: MessageAction) {
```

Comment:
Why is this method not private now?

Reply:
it's used in MessageActionsViewController. Before it was used in menu but handled with func forwardingTarget(for aSelector: Selector!) -> Any?